### PR TITLE
Rebranded the welcome page

### PR DESCRIPTION
**Pull request by MasterK0927**
Screenshot from 2024-03-25 12-27-06
Screenshot from 2024-03-25 12-27-44

1) Rebranded the welcome page.
2) Added the "-j8" flag into the ninja building tool's libs file, so that "llvm out of memory" error gets resolved.

### CODE REVIEW

File: chromium_src/ui/native_theme/native_theme.cc [hunk 1]
```diff
@@ -14,10 +14,9 @@
 namespace ui {
 
 SkColor NativeTheme::GetSystemButtonPressedColor(SkColor base_color) const {
-  const SkColor kBraveColorBrand = SkColorSetRGB(0xfb, 0x54, 0x2b);
+  const SkColor kBraveColorBrand =SkColorSetRGB(0xff, 0x54, 0x2b);
```
Comment (udaybansal19) [inline chromium_src/ui/native_theme/native_theme.cc]: what is this colour change for?

File: components/brave_welcome_ui/assets/brave_logo_3d@2x.webp
Comment (udaybansal19) [inline components/brave_welcome_ui/assets/brave_logo_3d@2x.webp:1]: what is this file for?

File: chromium_src/ui/native_theme/native_theme.cc (hunk 1 above)
Comment (MasterK0927) [reply chromium_src/ui/native_theme/native_theme.cc]: reverted to the default buttons color, and KbraveBrand color

File: components/brave_welcome_ui/assets/brave_logo_3d@2x.webp
Comment (MasterK0927) [reply components/brave_welcome_ui/assets/brave_logo_3d@2x.webp:1]: its the logo that is being displayed on the welcome page.

File: chromium_src/ui/native_theme/native_theme.cc (hunk 1 above)
Comment (udaybansal19) [reply chromium_src/ui/native_theme/native_theme.cc]: revert kara hai toh ye changed nahi ana chaiye na?

Comment (MasterK0927) [reply chromium_src/ui/native_theme/native_theme.cc]: mtlb bhaiya manually change kr diya hai, jo phele tha whi color mai

File: components/brave_welcome_ui/components/images/lion_logo.svg
Comment (udaybansal19) [inline components/brave_welcome_ui/components/images/lion_logo.svg:1]: is this logo being used on the welcome page or the other one?

File: chromium_src/ui/native_theme/native_theme.cc (hunk 1 above)
Comment (udaybansal19) [reply chromium_src/ui/native_theme/native_theme.cc]: manually bhi revert hua hai toh ye change nahi ana chaiye

File: components/brave_welcome_ui/components/images/lion_logo.svg
Comment (MasterK0927) [reply components/brave_welcome_ui/components/images/lion_logo.svg:1]: it is used in components/brave_welcome_ui/components/images/index.ts

File: chromium_src/ui/native_theme/native_theme.cc (hunk 1 above)
Comment (MasterK0927) [reply chromium_src/ui/native_theme/native_theme.cc]: okay bhaiya looking into it

Comment (MasterK0927) [reply chromium_src/ui/native_theme/native_theme.cc]: fixed it bhaiya, it was the typo

File: chromium_src/ui/native_theme/native_theme.cc [hunk 2]
```diff
@@ -14,10 +14,9 @@
 namespace ui {
 
 SkColor NativeTheme::GetSystemButtonPressedColor(SkColor base_color) const {
-  const SkColor kBraveColorBrand = SkColorSetRGB(0xfb, 0x54, 0x2b);
```
Comment (udaybansal19) [inline chromium_src/ui/native_theme/native_theme.cc]: remove extra changes from this file

File: components/brave_welcome_ui/components/images/lion_logo.svg
Comment (udaybansal19) [reply components/brave_welcome_ui/components/images/lion_logo.svg:1]: where is this logo being used and where is `brave_logo_3d@2x.webp` being used?

Comment (MasterK0927) [reply components/brave_welcome_ui/components/images/lion_logo.svg:1]: brave_logo_3d@2x.webp is being used in the brave_welcome_ui page

lion_logo.svg its use is still unknown but its defined in the images under brave_welcome_ui

File: chromium_src/ui/native_theme/native_theme.cc (hunk 2 above)
Comment (MasterK0927) [reply chromium_src/ui/native_theme/native_theme.cc]: done bhaiya

File: chromium_src/ui/native_theme/native_theme.cc [hunk 3]
```diff
@@ -19,5 +19,4 @@ SkColor NativeTheme::GetSystemButtonPressedColor(SkColor base_color) const {
   return color_utils::GetResultingPaintColor(
       SkColorSetA(kBraveColorBrand, is_dark ? 0x2b : 0x23), base_color);
 }
-
```
Comment (udaybansal19) [inline chromium_src/ui/native_theme/native_theme.cc]: yaar ye extra changes bhi hata do please. files ka indentation kharab nahi karte agar wahan kuch changes nahi hai toh

File: build/commands/lib/util.js [hunk 4]
```diff
@@ -1106,4 +1107,4 @@ const util = {
   },
 }
 
-module.exports = util
```
Comment (udaybansal19) [inline build/commands/lib/util.js]: same here as well

Comment (udaybansal19) [reply build/commands/lib/util.js]: yaar yahan pe extra line hat gayi. yahan pe koi changes nahi ane chaiye

Comment (MasterK0927) [reply build/commands/lib/util.js]: done bhaiya

File: chromium_src/ui/native_theme/native_theme.cc (hunk 3 above)
Comment (MasterK0927) [reply chromium_src/ui/native_theme/native_theme.cc]: done bhaiya

File: build/commands/lib/util.js (hunk 4 above)
Comment (udaybansal19) [reply build/commands/lib/util.js]: call?

Comment (udaybansal19) [reply build/commands/lib/util.js]: abhi bhi aa rha hai ye change

Comment (MasterK0927) [reply build/commands/lib/util.js]: > call?

okay bhaiya

Comment (udaybansal19) [reply build/commands/lib/util.js]: ussi link pe aa jao 
https://meet.google.com/tpy-hepz-kcd

File: build/commands/lib/util.js [hunk 5]
```diff
@@ -1106,4 +1106,4 @@ const util = {
...
\ No newline at end of file
+module.exports = util
```
Comment (udaybansal19) [inline build/commands/lib/util.js]: iss file mein koi changes hi nahi rehne chaiye

Comment (MasterK0927) [reply build/commands/lib/util.js]: Resolving

**Review by udaybansal19 — APPROVED** [review]

**Pull request merged**